p2p/pre2p/module.go

```diff
@@ -88,6 +99,12 @@ func (m *p2pModule) Start() error {
 		}
 	}()
 
+	m.
+		GetBus().
+		GetTelemetryModule().
+		GetTimeSeriesAgent().
+		CounterIncrement("p2p_nodes_online")
```

Comment:
Thoughts on moving the counter / gauge names into some sort of shared file with constants?

Reply:
That's the vision, hopefully json

Reply:
Not in this commit, I think these metrics might be subject to change and it wouldn't make sense to delay the PR for this addition at the moment.